libraries/lib-track/Track.cpp

```diff
@@ -51,7 +51,7 @@ Track::Track(const Track& orig, ProtectedCreationArg&&)
 void Track::Init(const Track &orig)
 {
    mId = orig.mId;
-
+   mProjectTempo = orig.mProjectTempo;
```

Comment:
Please edit the commit comment.  Put breaks in the very long line that is the third paragraph, and remove the last line.

Reply:
I reverted this change - wrong since `mProjectTempo` became part of `ChannelGroupData`.